crates/sui-protocol-config/src/lib.rs

```diff
@@ -1886,6 +1901,13 @@ impl ProtocolConfig {
                     cfg.consensus_max_transactions_in_block_bytes = Some(6 * 1_024 * 1024);
                     // 6 MB
                 }
+                37 => {
+                    if chain != Chain::Testnet && chain != Chain::Mainnet {
+                        cfg.feature_flags.enable_vdf = true;
+                        cfg.vdf_verify_vdf_cost = Some(2000);
```

Comment:
TODO: Add a meaningful cost. This function takes 130 ms to run on my machine for a 3072 bit discriminant.

Reply:
@dariorussi What do you think would be a reasonable gas price for this?

Reply:
I think @benr-ml had some ideas for pricing crypto functions

Reply:
yeah and sorry I just noticed this :( 
it seems the cost should be proportional to the allowed inputs, 130ms is a lot of time and we probably want a pretty high cost. 
Let's talk about this offline soon, please